alf/trainers/policy_trainer.py

```diff
@@ -138,7 +138,10 @@ def __init__(self, config: TrainerConfig, ddp_rank: int = -1):
     def train(self):
         """Perform training."""
         self._restore_checkpoint()
-        alf.summary.enable_summary()
+        if self._rank <= 0:
```

Comment:
In Algorithm._train_experience(), enable_summary() is also called, so this will not be effective.

Reply:
You may consider to change run_under_record_context so that _cond() always returns False.